src/caveexpress/server/map/Map.cpp

```diff
@@ -352,14 +352,16 @@ bool Map::isFailed () const
 	if (_players.empty())
 		return true;
 
+	/* old, bad
```

Comment:
please remove the commented code. if it is wrong is has no real value here. Even though I wonder how can we lose such a map now?

Reply:
Done